bump the patch ver

joyent-automation opened the pull request:
This PR shows that a review approval from a non-org member (@hamishsmith) does *not* satisfy the condition that one approving (i.e. user with write access to this repo) review is required.

trentm committed:
bump the patch ver

hamishsmith reviewed (approved):
Looks fine to me.

hamishsmith commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "play",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Perhaps 1.0.2 instead?

cburroughs replied:
This is totally something Trent would say!

hamishsmith reviewed (changes requested):
Actually please make a change (testing a review by non-org member that "Requests changes")

trentm committed:
bump 2 per feedback from hamish
Reviewed by: hamishsmith
Reviewed by: cburroughs <chris.burroughs@gmail.com>
Reviewed by: Tim Foster <tim.foster@joyent.com>

hamishsmith reviewed (dismissed):
Looks great now (testing "Approve" by non-org member)

cburroughs reviewed (dismissed):
TESTING

timfoster reviewed (dismissed):
this is fine

timfoster reviewed (dismissed):
ugh, still fine.

timfoster reviewed (approved):
yep, go for it

trentm commented:
... blah blah PR checklist ...

##### PR checklist

- [ ] tests added/updated
- [ ] testing redone on final commit
- [ ] related documentation updated
- [ ] got IA
